content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -68,6 +68,22 @@ or contain bugs.
 
 Skipped CSVs do not need to be present in a catalog or set of manifests prior to adding to a catalog.
 
+#### Example for replaces and skips
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: etcdoperator.v0.9.2
+  namespace: placeholder
+spec:
+    displayName: etcd
+    description: Etcd Operator
+    replaces: etcdoperator.v0.9.0
+    skips:
+    - etcdoperator.v0.9.1
```

Comment:
There are differing indentation levels between the `metadata` and `spec` properties.

Reply:
Ah! got it! I will modify that.

Reply:
Do we also want to add a description of this example as we do with the other recently introduced examples?

Reply:
Modified.